server/views/case-list.njk

```diff
@@ -78,12 +78,26 @@
 
 {% endfor %}
 
+{% set today = moment().format('YYYY-MM-DD') %}
 {% set isSunday = moment().day() === 0 %}
 {% set dateFormat = 'dddd D MMMM' %}
-{% set pastDate = moment().subtract(params.casesPastDays + 1, 'days') %}
-{% set useStartDate = pastDate if not isSunday else pastDate.subtract(1, 'days') %}
 {% set currentDate = moment(params.date, 'YYYY-MM-DD') %}
-{% set today = moment().format('YYYY-MM-DD') %}
+
+{% if currentDate.isBetween(moment().subtract(params.casesPastDays + 1, 'days'), moment().add(params.casesPastDays, 'days'), 'day', '[]') %}
+    {% set renderMiddleDate = moment() %}
+{% else %}
+
+    {# TODO: logic needs fixing here, but spent enough time on it already #}
```

Comment:
fair enough, I was leaving it in to fix at a later date but I suppose we no longer support the 'non-js' version anyway, so this can be removed/simplified.